app/presenters/document_export_presenter.rb

```diff
@@ -12,60 +12,127 @@ def as_json
+
+    topical_events.map do |te|
+      { id: te.id, content_id: te.content_id }
     end
+  end
 
-    provide_doctype_information(edition, output)
-    output[:government] = edition.government
-    output[:whitehall_admin_links] = resolve_whitehall_admin_links(edition.body)
-    if edition.withdrawn?
-      output[:whitehall_admin_links].concat(resolve_whitehall_admin_links(edition.unpublishing.explanation))
+  def complete_translations(edition)
+    translations = edition.try(:translations)
+    return [] unless translations
+
+    translations.map do |translation|
+      translation.attributes.except("edition_id").merge(
+        "public_base_path" => Whitehall.url_maker.public_document_path(edition, locale: translation.locale),
+      )
+    end
+  end
+
+  def stripped_edition_policies(edition)
+    edition_policies = edition.try(:edition_policies)
+    return [] unless edition_policies
+
+    edition_policies.map do |e_policy|
+      { id: e_policy.id, policy_content_id: e_policy.policy_content_id }
+    end
+  end
+
+  def complete_organisations(edition)
+    edition_organisations = edition.try(:edition_organisations)
+    return [] unless edition_organisations
+
+    edition_organisations.map do |e_org|
+      org = e_org.organisation
+      { id: e_org.id, content_id: org.content_id, lead: e_org.lead, lead_ordering: e_org.lead_ordering }
     end
-    output
   end
 
-  def complete_images_hash(edition_images)
-    edition_images.map do |image|
+  def stripped_authors(edition)
+    authors = edition.try(:authors)
+    return [] unless authors
+
+    authors.map do |a|
+      { id: a.id, uid: a.uid }
+    end
+  end
+
+  def stripped_specialist_sectors(edition)
+    sectors = edition.try(:specialist_sectors)
+    return [] unless sectors
+
+    sectors.map do |ss|
+      { id: ss.id, topic_content_id: ss.topic_content_id, primary: ss.primary }
+    end
+  end
+
+  def complete_images(edition)
+    images = edition.try(:images)
+    return [] unless images
+
+    images.map do |image|
       image.as_json(methods: :url)
     end
   end
 
-  def complete_attachments_hash(edition_attachments)
-    edition_attachments.map do |attachment|
+  def complete_attachments(edition)
+    attachments = edition.try(:attachments)
+    return [] unless attachments
+
+    attachments.map do |attachment|
       attachment.as_json(include: :attachment_data, methods: %i[url type]).tap do |json|
         json.merge!("govspeak_content" => attachment.govspeak_content.as_json) if attachment.respond_to?(:govspeak_content)
       end
     end
   end
 
-  def provide_doctype_information(edition, output)
-    %i[news_article_type publication_type corporate_information_page_type speech_type].each do |type|
-      output[type] = edition.public_send(type)&.key if edition.respond_to?(type)
+  def provide_doctype_information(edition)
+    %i[news_article_type publication_type corporate_information_page_type speech_type].each_with_object({}) do |type, memo|
```

Comment:
I'd suggest moving these out so they can be read and configured more easily:

```ruby
def doctypes
  %i[
    news_article_type
    publication_type
    corporate_information_page_type
    speech_type
  ]
end
```

Reply:
Not my code 😢But I agree with the premiss of your suggestion with a slight tweak, I'll make the list into constant array as this seems more ruby'esq.